ios/brave-ios/Sources/BraveVPN/Components/Region/BraveVPNRegionConfirmationView.swift

```diff
@@ -0,0 +1,131 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import BraveShared
+import BraveUI
+import Shared
+import SwiftUI
+
+public struct BraveVPNRegionConfirmationView: View {
+  let regionCountry: String?
+  let regionCity: String?
+  let regionCountryFlag: Image?
+
+  public init(regionCountry: String?, regionCity: String?, regionCountryISOCode: String?) {
+    self.regionCountry = regionCountry
+    self.regionCity = regionCity
+    self.regionCountryFlag = regionCountryISOCode?.regionFlag ?? Image(braveSystemName: "leo.globe")
+  }
+
+  public var body: some View {
+    VStack(spacing: 24) {
+      Image("region-change-banner", bundle: .module)
+        .resizable()
+        .frame(width: 96, height: 96)
+      Text(Strings.VPN.vpnRegionChangedTitle)
+        .multilineTextAlignment(.center)
+        .font(.headline)
+        .foregroundStyle(Color(braveSystemName: .textPrimary))
+      HStack {
+        if let regionCountryFlag = regionCountryFlag {
+          regionCountryFlag
+            .frame(width: 32, height: 32)
+            .padding(4)
+            .overlay(
+              RoundedRectangle(cornerRadius: 12.0, style: .continuous)
+                .strokeBorder(Color(.lightGray), lineWidth: 1.0)
+            )
+        }
+        VStack(alignment: .leading) {
+          if let regionCountry = regionCountry {
+            Text(regionCountry)
+              .font(.headline)
+              .foregroundStyle(Color(braveSystemName: .textPrimary))
+          }
+          if let regionCity = regionCity {
+            Text(regionCity)
+              .font(.subheadline)
+              .foregroundStyle(Color(braveSystemName: .textSecondary))
+          }
+        }
+        Spacer()
```

Comment:
prefer `.frame(maxWidth: .infinity, alignment: .leading)` on the `HStack` over trailing `Spacer()`

Reply:
yeah good tip fix at https://github.com/brave/brave-core/pull/24198/commits/db1d988f0db57643c18b4dd3cbca47d5c3adc98f